core/src/main/java/org/jdbi/v3/core/internal/JdbiClassUtils.java

```diff
@@ -81,4 +131,15 @@ public static Stream<Class<?>> superTypes(Class<?> type) {
 
         return result.stream();
     }
+
+    private static final Object[] NO_ARGS = new Object[0];
+
+    /**
+     * Safely move arguments passed from from a varargs call to a call that expects an array of objects.
+     * @param args A list of objects. May be null or empty.
+     * @return Returns an Array of objects. If the input was null, returns an empty array, otherwise all arguments as an array.
+     */
+    public static Object[] safeVarargs(Object... args) {
```

Comment:
This name doesn't quite make sense to me. If you accept varargs, then the array will never be null, right? The only way to get a null array would be to make a _non_-varargs invocation, and explicitly pass `null`.